lib/htmldump.js

```diff
@@ -130,6 +131,7 @@ function Dumper (articleChunkStream, options) {
     this.articles = [];
     this.waiters = [];
     this.done = false;
+    this.contentHost = 'http://' + options.prefix + '/api/rest_v1'
```

Comment:
I think it might make more sense to have `contentHost` as a command-line argument that replaces both `host` and `prefix`. That way one can specify exactly what they need. For example, outside of prod, that would be `https://en.wikipedia.org/api/rest_v1/` while if run inside, we can set it to `http://restbase.svc.eqiad.wmnet/en.wikipedia.org/v1/`